README.md

```diff
@@ -1,60 +1,66 @@
-[![Android Gems](http://www.android-gems.com/badge/skyfe79/AndroidChannel.svg?branch=master)](http://www.android-gems.com/lib/skyfe79/AndroidChannel)
+# OHOSChannel
 
-# AndroidChannel
+OHOSChannel is helper library for inter thread communication between main thread and worker thread. 
 
-[![Android Arsenal](https://img.shields.io/badge/Android%20Arsenal-AndroidChannel-green.svg?style=flat)](https://android-arsenal.com/details/1/2547)
+## Source
+Inspired by [skyfe79/AndroidChannel](https://github.com/skyfe79/AndroidChannel)
 
-AndroidChannel is helper library for inter thread communication between main thread and worker thread. AndroidChannel uses HandlerThread for inter thread communication.
+## Feature
+This library is helper library for inter thread communication between main thread and worker thread. 
+OHOSChannel uses HandlerThread for inter thread communication.
```

Comment:
Replace HandlerThread with HarmonyOS components that were used - EventHandler and EventRunner

Reply:
Done